Common/VR/PPSSPPVR.cpp

```diff
@@ -564,8 +564,6 @@ bool StartVRRender() {
 					M[11] -= separation.z;
 				}
 				memcpy(vrMatrix[matrix], M, sizeof(float) * 16);
-			} else {
-				assert(false);
```

Comment:
I realize cassert wasn't here, but personally I like using defensive asserts like this in case I forget to update a condition to add a new matrix case or etc. later.

-[Unknown]